libs/design/src/atoms/form/radioset/radioset.component.ts

```diff
@@ -0,0 +1,16 @@
+import { Component, OnInit, HostBinding, ElementRef, ViewChild, ContentChild, Input } from '@angular/core';
+
+@Component({
+  selector: 'daff-radio-set',
+  templateUrl: './radioset.component.html',
+  styleUrls: ['./radioset.component.scss']
```

Comment:
ChangeDetection.OnPush

Reply:
Addressed in 7eff94fc15e2ee0f2adf31c5de274475d7e0c73d